ocular/tests/single_node_chain_txs.rs

```diff
@@ -0,0 +1,355 @@
+/*
+ * Heavily borrows from https://github.com/cosmos/cosmos-rust/blob/main/cosmrs/tests/integration.rs
+ */
+
```

Comment:
Note: This test didn't run in this repo's PR tests (not sure why but it gets skipped, it passed locally however fwiw).

Since this requires docker up and running I'm not sure how pragmatic it is to have running on every PR vs just before we cut a release. If we only want the latter we can always run this locally as is, however if we want the former we need to add in docker support to our rust PR tests.